package.json

```diff
@@ -2,11 +2,11 @@
   "name": "shield-study-rappor",
   "version": "1.0.0",
   "description": "RAPPOR as a Shield Study",
-  "main": "index.js",
-  "engines" : {
-    "node" : ">=8.0"
+  "main": "addon/bootstrap.js",
+  "engines": {
+    "firefox": ">=45.0 <=*"
```

Comment:
Are you confident that's the minimum version? Are all the APIs you're using already there (crypto, etc.)?